packages/bot/src/listeners/guild/member/guildMemberAddSendSurvey.js

```diff
@@ -26,11 +28,13 @@ class GuildMemberAddSendSurvey extends Listener {
     try {
       const joinedUser = await client.users.fetch(member.id);
       logger.debug(`Sending new user message to: ${joinedUser.username}`);
-      await joinedUser.send({ embeds: [this.buildSurveyMessage(joinedUser)] });
+      await joinedUser.send({ embeds: [this.buildSurveyMessage(joinedUser, $api)] });
+      await $api.surveyLog.send(joinedUser.username, true, currentDate)
```

Comment:
Should this not be create?
```suggestion
      await $api.surveyLog.create(joinedUser.username, true, currentDate)
```